draft-ietf-sidrops-aspa-profile.xml

```diff
@@ -318,6 +318,9 @@
         For example, the number of Provider ASes that can be listed in a single RPKI-To-Router protocol ASPA PDU (<xref target="I-D.ietf-sidrops-8210bis" section="5.12"/>) is constrained to be 2^30-12.
         In addition to protocol limitations in the supply chain, locally defined restrictions could exist for the maximum file size a Relying Party implementation is willing to accept for a signed object.
       </t>
+      <t>
+        [Sriram comment] In the above, 2^30-12 seems strange! It is a huge number ~ 1 billion! What is the rationale. What does the -12 mean? Or is it a typo?
```

Comment:
Thanks. I understand. A mathematical purist might say it is (2^32 -1) instead of 2^30. Then subtract 12.  
So, 2^32 - 1 - 12 = 2^32 - 13. 
But no worries, This is just nitpicking.
May be put spaces before and after the minus, i.e., 2^30 - 12.  So, one does not mistake it for 2^(30-12).